### PR TITLE
feat: extract BillingService class

### PR DESCRIPTION
**Pull request by marie-j**
This will allow to share it between apps and modules

ref: MANAGER-4474
Signed-off-by: Marie JONES <marie.jones@corp.ovh.com>

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build-monorepository#9564.0 ✘  
* 
  * Build ✘

**Comment by FredericEspiau** [comment]
Any reason to use `api-models` instead of `models` ? I mean it's models for the front isn't it ? :)

**Comment by marie-j** [comment]
> Any reason to use `api-models` instead of `models` ? I mean it's models for the front isn't it ? :)

I was more focus on building the module than its name so I didn't put any extra thought to it but it makes sense to change

**Comment by FredericEspiau** [comment]
@marie-j that's what code reviews are for I guess !

File: packages/manager/modules/models/BillingService/BillingService.class.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ import head from 'lodash/head';
 import includes from 'lodash/includes';
 import isNull from 'lodash/isNull';
 import snakeCase from 'lodash/snakeCase';
+import 'moment';
```
Comment (FredericEspiau) [inline packages/manager/modules/models/BillingService/BillingService.class.js]: @cbourgois in lateral security position

File: yarn.lock [hunk 2]
```diff
@@ -3236,11 +3236,6 @@ ansi-regex@^2.0.0:
   resolved "https://registry.yarnpkg.com/ansi-regex/-/ansi-regex-2.1.1.tgz#c3b33ab5ee360d86e0e628f0468ae7ef27d654df"
   integrity sha1-w7M6te42DYbg5ijwRorn7yfWVN8=
 
-ansi-regex@^3.0.0:
```
Comment (FredericEspiau) [inline yarn.lock]: This is driving me crazy

Comment (marie-j) [reply yarn.lock]: We definitely need to check why we have this strange behaviour

File: packages/manager/modules/models/BillingService/BillingService.class.js (hunk 1 above)
Comment (marie-j) [reply packages/manager/modules/models/BillingService/BillingService.class.js]: @FredericEspiau we could think on a refactoring for this model later on, here the purpose is just to allow reusability

Comment (FredericEspiau) [reply packages/manager/modules/models/BillingService/BillingService.class.js]: Yeah it wasn't a criticism, you are entirely right to do this

**Comment by marie-j** [comment]
Splitted commits but they appear in the wrong order due to https://help.github.com/en/github/committing-changes-to-your-project/why-are-my-commits-in-the-wrong-order

**Pull request merged**